cedar-policy-validator/src/schema.rs

```diff
@@ -2943,9 +2948,38 @@ pub(crate) mod test {
                 "actions": { },
             }
         });
-        let schema = ValidatorSchema::from_json_value(src.clone(), Extensions::all_available());
-        assert_matches!(schema, Err(_));
+        assert_invalid_json_schema(src);
+
+        let src: serde_json::Value = json!({
+            "NS": {
+                "commonTypes": {
+                    "Record": {
```

Comment:
Maybe add a test case where the common type is never referenced.

Reply:
Done.